lib/class-wp-theme-json-gutenberg.php

```diff
@@ -331,6 +331,7 @@ class WP_Theme_JSON_Gutenberg {
 		'templateParts',
 		'title',
 		'version',
+		'behaviors',
```

Comment:
Didn't we move away from this?

Reply:
Yes, but in order to offer legacy support for the old syntax, we need to define it here as a valid value, otherwise it will get stripped out and we won't be able to read it.

Reply:
Did we ever ship the legacy syntax in WordPress Core?

Reply:
We have not

Reply:
Then I'm not sure it's necessary to keep the legacy support.